templates/template-for-tasks.rst

```diff
@@ -0,0 +1,171 @@
+:orphan: true
+	 
+.. Based on:
+   https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with
+   learnings from the 4 sfp pages built in branch DM-8717
+
+Please fill in the below sections with the contents as described in
+the reST comments, removing the comments as you do so, and using the
+links to locations in the `Guidance Document
+<writing-task-topics.html>`_ if needed.
+
+
+########################
+Structure of ExampleTask 
+########################
+
+.. Fill in introductory material here - this section needs the following:
+
+.. - Summary/context (1-2 sentences).
+
+.. - Concise summary of logic/algorithm in a paragaph and/or bullet list.
+
+.. - A sentence about each step, which can be either:
+
+..  a) A retargetable subtask
+
+..  b) A method within a task.
+
+`Guidance for the Introduction Section  <writing-task-topics.html#task-topics-intro>`_ .
+
+
+.. - Insert Module Membership here:
+
+..  This section needs only the module the task is implemented inside of.
+
+`Guidance for the Module Membership Section  <writing-task-topics.html#task-topics-module>`_ .
+
+.. Insert material for the SeeAlso Box here:
```

Comment:
Ok, i inserted the rst comment "Insert material for the SeeAlso Box here " after the directive, otherwise just compacted it down.